cmake/FindCVC5PythonicAPI.cmake

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# Top contributors (to current version):
+#   Gereon Kremer, Aina Niemetz
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# Find cvc5 pythonic api. 
+# CVC5PythonicAPI_FOUND - found cvc5 pythonic api
+# CVC5PythonicAPI_BASEDIR - the base directory of the cvc5 pythonic api
+##
+
+include(deps-helper)
+
+check_ep_downloaded("CVC5PythonicAPI")
+if(NOT CVC5PythonicAPI_DOWNLOADED)
+  check_auto_download("CVC5PythonicAPI" "--no-python-bindings")
+endif()
+
+include(ExternalProject)
+
+ExternalProject_Add(
+  CVC5PythonicAPI
+  ${COMMON_EP_CONFIG}
+  GIT_REPOSITORY https://github.com/cvc5/cvc5_z3py_compat.git
+  GIT_TAG main
```

Comment:
Do we really want to always pull main instead of a commit? I'd suggest to use a fixed commit as otherwise it will be hard to debug errors if the pythonic API can change anytime.

Reply:
The pythonic API has no installation mechanism by itself (right now).